services/orders_item_add/lambda_function.py

```diff
@@ -28,8 +28,11 @@ def lambda_handler(event, context):
 
         response = orders_table.update_item(
             Key={'id': order_id},
-            UpdateExpression="set #tms = :items",
-            ExpressionAttributeValues={':items': order_items},
+            UpdateExpression="SET #tms = :items, total_cost + :cost",
```

Comment:
this code generates an error on AWS, I don't think this is correct. Did you really test your code out on the cloud ?

Reply:
The comma should be removed here

Reply:
@preppas Is it an idea to put these changes in a separate PR?

Reply:
No, don't put the changes in a separate PR, this will mean we're pushing faililing code on master, which you should never do !

Reply:
I am referring to the commits of @preppas which fix the issue documented in #12 
as opposed to implementing the payment service. Since the bug is already in the master, I don't think it matters.

Reply:
This is not a bug on master ! It's just functionality that was not yet implemented, but deploying the stack as is on the master and calling the lambda function will not trigger a 400 response and a failure of the lambda (big difference) 😄

Reply:
You are misunderstanding me. I am proposing to move all of @preppas commits to a separate PR (which includes this bug) since they are not directly related to #4 (instead #12). Without these commits, this PR works as advertised as well.

Reply:
Moved them to #22